stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -16,8 +16,11 @@ import Swift
 
 /// Starts a new scope that can contain a dynamic number of child tasks.
 ///
-/// A group waits for all of its child tasks
-/// to complete or be canceled before it returns.
+/// A group *always* waits for all of its child tasks
+/// to complete before it returns. Even cancelled tasks must run until
```

Comment:
Apple Style for prose and API style for symbols spell this opposite ways:

```suggestion
/// to complete before it returns. Even canceled tasks must run until
```

Reply:
Applied to all "cancelled" in this file, thank you